### ISSUE TITLE
Fix logout

### ISSUE THREAD

**Issue by marshallswain** [canjs/can-connect-feathers#40]
can-connect-feathers should probably do something more like this for destroyData:

```js
destroyData: function (session) {
  return feathersClient.logout().then(function(){
    Session.trigger('destroyed', [session]);
    this.destroyInstance(session);
  }.bind(this));
}
```

Then the "session destroyed" event will fire and clear the session data from the UI.

**Comment by marshallswain** [comment]
Related to https://github.com/donejs/bitcentive/issues/175

**Comment by marshallswain** [comment]
This is fixed in the currently-published `3.0.0-pre.22`.
